src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -903,39 +903,89 @@ public File createTmpDir() throws IOException {
         return env.temporaryDirectoryAllocator.allocate();
     }
 
+    // TODO replace return types once the Jenkins core is updated
```

Comment:
Do you really foresee such a class rename in core? Seems like a lot of disruption for a cosmetic issue.

Reply:
I do not expect it anytime soon TBH. Depends on contributions though